components/chassis.py

```diff
@@ -17,10 +19,40 @@ class Chassis:
     drive_motor_d = CANTalon
     sd = NetworkTable
 
+    # some calculatins that provide numbers used by the motion profiling
+    wheel_circumference = 6*0.0254*math.pi # m
+    counts_per_revolution = 1024
+    # convert from meters to sensor units
+    counts_per_meter = counts_per_revolution/wheel_circumference
+
+    # to convert from m/s to counts per 100ms
+    velocity_to_native_units = 0.1*counts_per_meter
+
+    # max velocity as measured by talons driving flat out
+    max_vel_native = 800 # ticks / 100ms
+    # convert to SI units - m/s
+    max_vel = (10*max_vel_native*wheel_circumference)/counts_per_revolution
+    max_acc = 1 # m/s
+
+    wheelbase_width = 0.629666 # m
+
+    pid_profile = {
+            "kP" : 3,
+            "kI" : 0,
+            "kD" : 0,
+            "kF" : int(1023/max_vel_native),
+            "ramp-rate" : 36 # change in volts, in v/sec
+            }
```

Comment:
Indentation is a bit wonky here. (Yes, I know Vim's Python `indentexpr` is a bit retarded. It annoys me to no end. Maybe I could go off and fix it…)

Also, nit: no whitespace before a colon.